src/app/profiles/profiles.editor.modal.ts

```diff
@@ -274,31 +241,20 @@ export class ProfilesEditorModalComponent {
             setup: editor => {
                 self.editor = editor;
                 editor.on('Change', () => {
-                    self.data.profile.signature = editor.getContent();
+                    self.identity.signature = editor.getContent();
```

Comment:
I believe the new lambdas (fat arrows) preserve `this`, so it isn't necessary to alias `this` to `self`

Reply:
Arrow functions don't have their own [bindings](https://developer.mozilla.org/en-US/docs/Glossary/Binding) to [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this), [arguments](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments), or [super](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/super), and should not be used as [methods](https://developer.mozilla.org/en-US/docs/Glossary/Method).

Source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Lexical_this

Reply:
ah another case of the codebase doing the same thing 2 different ways... compose doesnt bother with self. so have removed it here too..